.gitlab.mk

```diff
@@ -117,8 +117,11 @@ vms_shutdown:
 GIT_DESCRIBE=$(shell git describe HEAD)
 GIT_TAG=$(shell git tag --points-at HEAD)
 MAJOR_VERSION=$(word 1,$(subst ., ,$(GIT_DESCRIBE)))
-BUCKET="series-$(MAJOR_VERSION)"
+TNT_VERSION_S3_DIR=series-$(MAJOR_VERSION)
 S3_BUCKET_URL="s3://tarantool_repo/sources"
+RWS_BASE_URL=https://rws.tarantool.org
+RWS_ENDPOINT=${RWS_BASE_URL}/${REPO_S3_DIR}/${TNT_VERSION_S3_DIR}/${OS}/${DIST}
```

Comment:
I would fail if `REPO_S3_DIR` is not provided. Just in case.

Reply:
Done.